docs/permissions.md

```diff
@@ -43,3 +43,10 @@ The following table depicts the various user permission levels in a project.
 | Add/Remove labels of helm chart version |       | ✓         | ✓      | ✓             |
 | See a list of project robots            |       |           | ✓      | ✓             |
 | Create/edit/delete project robots       |       |           |        | ✓             |
+| See configured CVE whitelist            | ✓     | ✓         | ✓      | ✓             |
+| Create/edit/remove CVE whitelist        |       |           | ✓      | ✓             |
+| Enable/disable webhooks                 |       | ✓         | ✓      | ✓             |
+| Create/delete tag retention rules       |       | ✓         | ✓      | ✓             |
+| Enable/disable tag retention rules      |       | ✓         | ✓      | ✓             |
+| See quotas set for project              | ✓     | ✓         | ✓      | ✓             |
+| Edit quotas for new project             |       |           |        | ✓             |
```

Comment:
Only system administrator can edit quotas for project.

Reply:
so project admins can't edit quota?